### DISCUSSION TITLE
Round end ints converted?

### DISCUSSION THREAD

**Discussion by xGuysOG** [saul/demofile-net#79] [Q&A] [answered]
Hi everyone, do we have a converted string or table over what the ints for winner and Reason are here?
`public class Source1RoundEndEvent : Source1GameEventBase
{
    public override string GameEventName => "round_end";

    public int Winner { get; set; }

    public int Reason { get; set; }

    public string Message { get; set; } = "";

    public int Legacy { get; set; }

    public int PlayerCount { get; set; }

    public int Nomusic { get; set; }

    internal Source1RoundEndEvent(DemoParser demo)
        : base(demo)
    {
    }
}`

**Comment by in0finite** [comment, marked as answer]
Yes, `Winner` should correspond to `GameRules.RoundWinStatus` (which represents enum `CSTeamNumber`), `Reason` should correspond to `GameRules.RoundWinReason` (which represents enum `CSRoundEndReason`).
